api/python/t4/packages.py

```diff
@@ -527,6 +527,13 @@ def push(self, name, path):
             A new package that points to the copied objects
         """
         dest = fix_url(path)
+        if dest.startswith('file://'):
+            # local push
+            os.makedirs(path, exist_ok=True)
+            pkg = self._materialize(dest)
```

Comment:
when materializing to s3, we splat it into `{dest}/{pkg_name}` we probably should do the same for local disk too.